_includes/category-list-item.html

```diff
@@ -1,5 +1,5 @@
   <div class="post {% if forloop.index == 1 %}  first_post {% endif %}">
-  <div class="post_img" style="background-image:url({{ site.url }}{{ site.baseurl }}assets/images/blogcover/{% if forloop.index == 1 %}full{%else%}mini{% endif %}/{{ post.cover }});"></div>
+  <div class="post_img" style='background-image:url({% asset blogcover/"{{ post.cover }}" srcset:width="1300 2x" srcset:width="650 1x" @path %});'></div>

```

Comment:
This is the correct width, right? Not the one that I added to the blog post.

Reply:
@vpetersson I used the settings as for blog images. What parameters do you suggest to use?

Reply:
I used the settings as for blog images. What parameters do you suggest to use?

Reply:
![Screen Shot 2019-07-18 at 2 55 05 PM](https://user-images.githubusercontent.com/357664/61463674-d49afd00-a96c-11e9-82fc-dc8063d2285a.jpg)

Reply:
Inspect the div and see how wide these objects are. It's defined in the code so it shouldn't really be an unknown.